### ISSUE TITLE
Can't build « debian » package...

### ISSUE THREAD

**Issue by Creteil** [amit9838/mousam#139]
Trying to build « debian » package fail with error :

```
bigbob@bigbob-t480s:~/tmp/Building/mousam/mousam$ LANG=C rm debian/*~ ; dpkg-checkbuilddeps && LANG=C dpkg-buildpackage -rfakeroot
rm: cannot remove 'debian/*~': No such file or directory
dpkg-buildpackage: info: source package mousam
dpkg-buildpackage: info: source version 1.3.1-2-g96217e0
dpkg-buildpackage: info: source distribution UNRELEASED
dpkg-buildpackage: info: source changed by COLIN Stephane <bigbob.fun@gmail.com>
dpkg-buildpackage: info: host architecture amd64
 dpkg-source --before-build .
 debian/rules clean
dh clean --with=python3 --buildsystem=meson
   dh_auto_clean -O--buildsystem=meson
   dh_autoreconf_clean -O--buildsystem=meson
   dh_clean -O--buildsystem=meson
 dpkg-source -b .
dpkg-source: warning: native package version may not have a revision
dpkg-source: info: using source format '3.0 (native)'
dpkg-source: info: building mousam in mousam_1.3.1-2-g96217e0.tar.xz
dpkg-source: info: building mousam in mousam_1.3.1-2-g96217e0.dsc
 debian/rules binary
dh binary --with=python3 --buildsystem=meson
   dh_update_autotools_config -O--buildsystem=meson
   dh_autoreconf -O--buildsystem=meson
   dh_auto_configure -O--buildsystem=meson
	cd obj-x86_64-linux-gnu && DEB_PYTHON_INSTALL_LAYOUT=deb LC_ALL=C.UTF-8 meson setup .. --wrap-mode=nodownload --buildtype=plain --prefix=/usr --sysconfdir=/etc --localstatedir=/var --libdir=lib/x86_64-linux-gnu -Dpython.bytecompile=-1
The Meson build system
Version: 1.2.3
Source dir: /home/bigbob/tmp/Building/mousam/mousam
Build dir: /home/bigbob/tmp/Building/mousam/mousam/obj-x86_64-linux-gnu
Build type: native build
Project name: mousam
Project version: 1.3.1
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program msgfmt found: YES (/usr/bin/msgfmt)
Program desktop-file-validate found: YES (/usr/bin/desktop-file-validate)
Program appstream-util found: YES (/usr/bin/appstream-util)
Program glib-compile-schemas found: YES (/usr/bin/glib-compile-schemas)
Found pkg-config: /usr/bin/pkg-config (0.29.2)
Build-time dependency gio-2.0 found: YES 2.72.4
Program /usr/bin/glib-compile-resources found: YES (/usr/bin/glib-compile-resources)
Program python3 found: YES (/usr/bin/python3)
Configuring mousam using configuration
Configuring constants.py using configuration
Program msginit found: YES (/usr/bin/msginit)
Program msgmerge found: YES (/usr/bin/msgmerge)
Program xgettext found: YES (/usr/bin/xgettext)
Dependency gio-2.0 found: YES 2.72.4 (cached)
Program /usr/bin/glib-compile-schemas found: YES (/usr/bin/glib-compile-schemas)
Program gtk4-update-icon-cache found: YES (/usr/bin/gtk4-update-icon-cache)
Program update-desktop-database found: YES (/usr/bin/update-desktop-database)
Build targets in project: 18

mousam 1.3.1

  User defined options
    buildtype         : plain
    libdir            : lib/x86_64-linux-gnu
    localstatedir     : /var
    prefix            : /usr
    sysconfdir        : /etc
    wrap_mode         : nodownload
    python.bytecompile: -1

Found ninja-1.10.1 at /usr/bin/ninja
   dh_auto_build -O--buildsystem=meson
	cd obj-x86_64-linux-gnu && LC_ALL=C.UTF-8 ninja -j8 -v
[1/15] /usr/bin/msgfmt -o po/es/LC_MESSAGES/mousam.mo ../po/es.po
[2/15] /usr/bin/msgfmt -o po/ko/LC_MESSAGES/mousam.mo ../po/ko.po
[3/15] /usr/bin/msgfmt -o po/fr/LC_MESSAGES/mousam.mo ../po/fr.po
[4/15] /usr/bin/msgfmt -o po/de/LC_MESSAGES/mousam.mo ../po/de.po
[5/15] /usr/bin/msgfmt -o po/hi/LC_MESSAGES/mousam.mo ../po/hi.po
[6/15] /usr/bin/glib-compile-resources ../src/mousam.gresource.xml --sourcedir ../src --internal --generate --target src/mousam.gresource --dependency-file src/mousam.gresource.d
[7/15] /usr/bin/msgfmt -o po/nl/LC_MESSAGES/mousam.mo ../po/nl.po
[8/15] /usr/bin/msgfmt -o po/ro/LC_MESSAGES/mousam.mo ../po/ro.po
[9/15] /usr/bin/msgfmt -o po/tr/LC_MESSAGES/mousam.mo ../po/tr.po
[10/15] /usr/bin/msgfmt -o po/ru/LC_MESSAGES/mousam.mo ../po/ru.po
[11/15] /usr/bin/msgfmt -o po/pt_BR/LC_MESSAGES/mousam.mo ../po/pt_BR.po
[12/15] /usr/bin/msgfmt -o po/pl/LC_MESSAGES/mousam.mo ../po/pl.po
[13/15] /usr/bin/msgfmt -o po/uk/LC_MESSAGES/mousam.mo ../po/uk.po
[14/15] /usr/bin/meson --internal msgfmthelper --msgfmt=/usr/bin/msgfmt ../data/io.github.amit9838.mousam.desktop.in data/io.github.amit9838.mousam.desktop desktop ../data/../po
[15/15] /usr/bin/meson --internal msgfmthelper --msgfmt=/usr/bin/msgfmt ../data/io.github.amit9838.mousam.appdata.xml.in data/io.github.amit9838.mousam.appdata.xml xml ../data/../po
   dh_auto_test -O--buildsystem=meson
	cd obj-x86_64-linux-gnu && DEB_PYTHON_INSTALL_LAYOUT=deb LC_ALL=C.UTF-8 MESON_TESTTHREADS=8 meson test
ninja: Entering directory `/home/bigbob/tmp/Building/mousam/mousam/obj-x86_64-linux-gnu'
ninja: no work to do.
1/3 Validate desktop file          OK              0.03s
2/3 Validate schema file           OK              0.01s
3/3 Validate appstream file        FAIL            3.06s   exit status 1
>>> MALLOC_PERTURB_=141 /usr/bin/appstream-util validate data/io.github.amit9838.mousam.appdata.xml


Ok:                 2   
Expected Fail:      0   
Fail:               1   
Unexpected Pass:    0   
Skipped:            0   
Timeout:            0   

Full log written to /home/bigbob/tmp/Building/mousam/mousam/obj-x86_64-linux-gnu/meson-logs/testlog.txt
	cd obj-x86_64-linux-gnu && tail -v -n \+0 meson-logs/testlog.txt
==> meson-logs/testlog.txt <==
Log of Meson test suite run on 2024-06-18T10:38:31.917571

Inherited environment: COMPIZ_BIN_PATH=/usr/bin/ DEB_HOST_MULTIARCH=x86_64-linux-gnu DEB_HOST_GNU_SYSTEM=linux-gnu SSH_AGENT_LAUNCHER=gnome-keyring DEB_BUILD_GNU_TYPE=x86_64-linux-gnu DEB_TARGET_ARCH_LIBC=gnu DEB_BUILD_ARCH_ENDIAN=little MANDATORY_PATH=/usr/share/gconf/unity.mandatory.path DEB_BUILD_GNU_SYSTEM=linux-gnu DEB_BUILD_ARCH_BITS=64 DEB_BUILD_OPTIONS=parallel=8 LIBVIRT_DEFAULT_URI=qemu:///system GDM_LANG=fr_FR GTK3_MODULES=xapp-gtk3-module QT_ACCESSIBILITY=1 DEB_HOST_ARCH=amd64 DEB_TARGET_ARCH_ENDIAN=little DEB_BUILD_PROFILES=noudeb GTK2_MODULES=overlay-scrollbar PWD=/home/bigbob/tmp/Building/mousam/mousam SOURCE_DATE_EPOCH=1718658903 DPKG_BUILDPACKAGE_PACKAGE_VERSION=1.3.1-2-g96217e0 OLDPWD=/home/bigbob XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session0 DEB_HOST_ARCH_BITS=64 LANG=C XAUTHORITY=/home/bigbob/.Xauthority GNOME_TERMINAL_SERVICE=:1.146 MAKEFLAGS='' DEB_TARGET_ARCH=amd64 DEB_HOST_GNU_CPU=x86_64 DEB_BUILD_ARCH_LIBC=gnu DPKG_BUILDPACKAGE_PACKAGE_NAME=mousam LESSOPEN='| /usr/bin/lesspipe %s' MFLAGS='' SSH_AUTH_SOCK=/run/user/1000/keyring/ssh DEB_TARGET_ARCH_BITS=64 XDG_SESSION_DESKTOP=unity DEB_BUILD_ARCH_CPU=amd64 XDG_SESSION_TYPE=x11 DPKG_BUILDPACKAGE_PACKAGE_ARCH=amd64 LD_PRELOAD=libgtk3-nocsd.so.0 DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus DPKG_BUILDPACKAGE_OS_RELEASE_VERSION_ID=22.04 DEB_HOST_ARCH_OS=linux GNOME_DESKTOP_SESSION_ID=this-is-deprecated SHELL=/bin/bash XMODIFIERS=@im=ibus DOTNET_BUNDLE_EXTRACT_BASE_DIR=/home/bigbob/.cache/dotnet_bundle_extract DPKG_BUILDPACKAGE_OS_RELEASE_ID=ubuntu SHLVL=1 DEB_HOST_ARCH_CPU=amd64 GNOME_TERMINAL_SCREEN=/org/gnome/Terminal/screen/e9560183_3393_4fdd_9870_453671f00bdb PATH=/home/bigbob/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/home/bigbob/.dotnet/tools DEB_BUILD_ARCH_ABI=base LANGUAGE=fr_FR DEB_BUILD_ARCH=amd64 DEB_HOST_ARCH_LIBC=gnu COMPIZ_CONFIG_PROFILE=ubuntu DEB_TARGET_ARCH_CPU=amd64 LESSCLOSE='/usr/bin/lesspipe %s %s' GTK_CSD=0 DEB_HOST_GNU_TYPE=x86_64-linux-gnu GTK_IM_MODULE=ibus XDG_CURRENT_DESKTOP=Unity:Unity7:ubuntu LS_COLORS='rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:' XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0 DESKTOP_SESSION=unity DEB_BUILD_ARCH_OS=linux CLUTTER_IM_MODULE=ibus LOGNAME=bigbob SYSTEMD_EXEC_PID=3671 DEB_TARGET_MULTIARCH=x86_64-linux-gnu DEB_BUILD_GNU_CPU=x86_64 DEB_TARGET_ARCH_OS=linux DISPLAY=:0 GTK_MODULES=gail:atk-bridge:unity-gtk-module USER=bigbob DEFAULTS_PATH=/usr/share/gconf/unity.default.path DEB_HOST_ARCH_ABI=base DEB_BUILD_MULTIARCH=x86_64-linux-gnu DEB_TARGET_GNU_TYPE=x86_64-linux-gnu DEB_TARGET_ARCH_ABI=base _=/usr/bin/dpkg-buildpackage MAKE_TERMOUT=/dev/pts/3 UNITY_DEFAULT_PROFILE=unity GPG_AGENT_INFO=/run/user/1000/gnupg/S.gpg-agent:0:1 COLORTERM=truecolor DEB_TARGET_GNU_CPU=x86_64 MAKE_TERMERR=/dev/pts/3 XDG_SESSION_CLASS=user HOME=/home/bigbob/tmp/Building/mousam/mousam/debian/.debhelper/generated/_source/home QT_IM_MODULE=ibus DEB_RULES_REQUIRES_ROOT=no DEB_TARGET_GNU_SYSTEM=linux-gnu DEB_HOST_ARCH_ENDIAN=little TERM=xterm-256color IM_CONFIG_PHASE=1 VTE_VERSION=6800 XDG_GREETER_DATA_DIR=/var/lib/lightdm-data/bigbob UNITY_HAS_3D_SUPPORT=true GDMSESSION=unity MAKELEVEL=1 DH_INTERNAL_BUILDFLAGS=1 ASFLAGS='' ASFLAGS_FOR_BUILD='' CFLAGS='-g -O2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ffile-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=. -flto=auto -ffat-lto-objects -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection -fdebug-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=/usr/src/mousam-1.3.1-2-g96217e0' CFLAGS_FOR_BUILD='-g -O2' CPPFLAGS='-Wdate-time -D_FORTIFY_SOURCE=3' CPPFLAGS_FOR_BUILD='' CXXFLAGS='-g -O2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ffile-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=. -flto=auto -ffat-lto-objects -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection -fdebug-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=/usr/src/mousam-1.3.1-2-g96217e0' CXXFLAGS_FOR_BUILD='-g -O2' DFLAGS=-frelease DFLAGS_FOR_BUILD=-frelease FCFLAGS='-g -O2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ffile-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=. -flto=auto -ffat-lto-objects -fstack-protector-strong -fstack-clash-protection -fcf-protection -fdebug-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=/usr/src/mousam-1.3.1-2-g96217e0' FCFLAGS_FOR_BUILD='-g -O2' FFLAGS='-g -O2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ffile-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=. -flto=auto -ffat-lto-objects -fstack-protector-strong -fstack-clash-protection -fcf-protection -fdebug-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=/usr/src/mousam-1.3.1-2-g96217e0' FFLAGS_FOR_BUILD='-g -O2' LDFLAGS='-Wl,-Bsymbolic-functions -flto=auto -ffat-lto-objects -Wl,-z,relro -specs=/usr/share/dpkg/elf-package-metadata.specs' LDFLAGS_FOR_BUILD='' OBJCFLAGS='-g -O2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ffile-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=. -flto=auto -ffat-lto-objects -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection -fdebug-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=/usr/src/mousam-1.3.1-2-g96217e0' OBJCFLAGS_FOR_BUILD='-g -O2' OBJCXXFLAGS='-g -O2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ffile-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=. -flto=auto -ffat-lto-objects -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection -fdebug-prefix-map=/home/bigbob/tmp/Building/mousam/mousam=/usr/src/mousam-1.3.1-2-g96217e0' OBJCXXFLAGS_FOR_BUILD='-g -O2' XDG_RUNTIME_DIR=/tmp/dh-xdg-rundir-h5yfGbsm DEB_PYTHON_INSTALL_LAYOUT=deb LC_ALL=C.UTF-8 MESON_TESTTHREADS=8 

==================================== 1/3 =====================================
test:         Validate desktop file
start time:   08:38:31
duration:     0.03s
result:       exit status 0
command:      MALLOC_PERTURB_=213 /usr/bin/desktop-file-validate data/io.github.amit9838.mousam.desktop
----------------------------------- stdout -----------------------------------
data/io.github.amit9838.mousam.desktop: warning: value "Application;Utility;GTK;" for key "Categories" in group "Desktop Entry" contains a deprecated value "Application"
==============================================================================

==================================== 2/3 =====================================
test:         Validate schema file
start time:   08:38:31
duration:     0.01s
result:       exit status 0
command:      MALLOC_PERTURB_=124 /usr/bin/glib-compile-schemas --strict --dry-run /home/bigbob/tmp/Building/mousam/mousam/data
==============================================================================

==================================== 3/3 =====================================
test:         Validate appstream file
start time:   08:38:31
duration:     3.06s
result:       exit status 1
command:      MALLOC_PERTURB_=141 /usr/bin/appstream-util validate data/io.github.amit9838.mousam.appdata.xml
----------------------------------- stdout -----------------------------------
data/io.github.amit9838.mousam.appdata.xml: ÉCHEC:
• tag-invalid           : <url> type invalid [unknown]
• attribute-invalid     : <screenshot> height too large [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss5-overcast.png] maximum is 900px
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss5-overcast.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss5-overcast.png]
• attribute-invalid     : <screenshot> height too large [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss3-clear_sky-imperial.png] maximum is 900px
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss3-clear_sky-imperial.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss3-clear_sky-imperial.png]
• attribute-invalid     : <screenshot> height too large [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss4-light_mode.png] maximum is 900px
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss4-light_mode.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss4-light_mode.png]
• attribute-invalid     : <screenshot> height too large [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss1-dark_mode.png] maximum is 900px
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss1-dark_mode.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss1-dark_mode.png]
• attribute-invalid     : <screenshot> height too large [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss2-thunderstorm.png] maximum is 900px
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss2-thunderstorm.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/amit9838/mousam/master/screenshots/ss2-thunderstorm.png]
• style-invalid         : <caption> is too short [Orage];shortest allowed is 10 chars
• style-invalid         : <ul> cannot start a description [(null)]
• style-invalid         : Not enough <p> tags for a good description [0/1]
----------------------------------- stderr -----------------------------------
La validation des fichiers a échoué
==============================================================================


Summary of Failures:

3/3 Validate appstream file FAIL            3.06s   exit status 1

Ok:                 2   
Expected Fail:      0   
Fail:               1   
Unexpected Pass:    0   
Skipped:            0   
Timeout:            0   
dh_auto_test: error: cd obj-x86_64-linux-gnu && DEB_PYTHON_INSTALL_LAYOUT=deb LC_ALL=C.UTF-8 MESON_TESTTHREADS=8 meson test returned exit code 1
make: *** [debian/rules:8: binary] Error 255
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
bigbob@bigbob-t480s:~/tmp/Building/mousam/mousam$
```

Does someone have any idea ?

**Comment by Creteil** [comment]
OK, fixed it by using « validate-relax » in place of « validate » :

```
--- data/meson.build~   2024-06-18 10:58:41.000000000 +0200
+++ data/meson.build    2024-06-18 10:59:05.490047760 +0200
@@ -22,7 +22,7 @@
 
 appstream_util = find_program('appstream-util', required: false)
 if appstream_util.found()
-  test('Validate appstream file', appstream_util, args: ['validate', appstream_file])
+  test('Validate appstream file', appstream_util, args: ['validate-relax', appstream_file])
 endif
```

Hope it help...

**Comment by amit9838** [comment]
@hsbasu is our active maintainer for debian packages, you may tag him for debian related ssues.

**Comment by Creteil** [comment]
What do you mean by _**« you may tag him for debian related issues. »**_ ?

**Comment by amit9838** [comment]
I mean you can tag him with the question like this "@hsbasu" , then he will be notified with the issue.

**Comment by Creteil** [comment]
Ah OK, look like you already invoked him with your example, so let's wait for him now...

**Comment by hsbasu** [comment]
The best way to invoke debian packaging is:
```
git clone https://github.com/amit9838/mousam.git
cd mousam
dpkg-buildpackage -us -uc
```

**Comment by Creteil** [comment]
Your answer doesn't make sense, the 2 builds are exactly the same :

```
bigbob@bigbob-t480s:~/tmp/Building/mousam$ diff mousam_1.3.1-2-g96217e0_all.deb /srv/local-apt-repository/mousam_1.3.1-2-g96217e0_all.deb 
bigbob@bigbob-t480s:~/tmp/Building/mousam$
```

**Comment by Creteil** [comment]
By the way, like I say it is already fixed in comment [#2](https://github.com/amit9838/mousam/issues/139#issuecomment-2175580283)...

**Comment by hsbasu** [comment]
I don't know how you are building it, but it builds fine on my system and on [Launchpad](https://code.launchpad.net/~mamolinux/+recipe/mousam-unstable) without the patch in [#2](https://github.com/amit9838/mousam/issues/139#issuecomment-2175580283).
